LDAR_Sim/model_code/methods/deployment/orbit_company.py

```diff
@@ -0,0 +1,56 @@
+from methods.deployment._base import SchedCompany as BaseSchedCompany
+
+
+class Schedule(BaseSchedCompany):
+    def __init__(self, config, parameters, state):
+        self.parameters = parameters
+        self.config = config
+        self.state = state
+
+        # --- inherited ---
+        # base.company ->  get_deployment_dates()
+        # base.company ->  can_deploy_today()
+
+    def get_deployment_dates(self):
```

Comment:
This is already part of the base .company :
`  --- inherited ---`
 `base.company ->  get_deployment_dates()`
` base.company ->  can_deploy_today()`